server/priv/marketing/blog/2025/07/08/server-fcl.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Tuist Server Source Code Is Now Public"
```

Comment:
```suggestion
title: "Tuist Server is Now Source Available"
```

I would use source available over "public" – I think it's something folks are quite familiar with (and we use that in the rest of the blog post)